std_msgs/Time timeTag
int16 arucoId
geometry_msgs/Point32[] points 